Patch to add Solaris 12 support.  This patch has been submitted
upstream.

--- puppetlabs-concat-1.2.1/metadata.json.orig	2015-04-17 09:08:46.891261878 -0600
+++ puppetlabs-concat-1.2.1/metadata.json	2015-04-17 09:08:55.480888584 -0600
@@ -67,7 +67,8 @@
       "operatingsystem": "Solaris",
       "operatingsystemrelease": [
         "10",
-        "11"
+        "11",
+        "12"
       ]
     },
     {
